utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	errs "github.com/ONSdigital/dp-import-api/apierrors"
+	"strconv"
+)
+
+// ValidatePositiveInt obtains the positive int value of query var defined by the provided varKey
+func ValidatePositiveInt(parameter string) (val int, err error) {
+
+	val, err = strconv.Atoi(parameter)
+	if err != nil {
+		return -1, errs.ErrInvalidQueryParameter
```

Comment:
It is the same for this error, I wasn't sure if we wanted to standardise the error responses but this is what has been used in the permissions-api: https://github.com/ONSdigital/dp-permissions-api/blob/develop/apierrors/errors.go#L12